### ISSUE TITLE
Dark Mode Dropdown Color Contrast

### ISSUE THREAD

**Issue by atyazin** [deso-protocol/frontend#122]
Currently, the dropdown is the same color as the background layer.  The UI would benefit from increased contrast between overlayed layers and background layers (i.e. exposed dropdowns).

I'd like to request/suggest that the color for dropdown.menu either includes a brightness filter of 112% or an outright color change to #2E2E2E. 

Additionally, including a drop shadow will also increase contrast. My suggestion is "box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.25);"

![image](https://user-images.githubusercontent.com/69254112/121586130-e9695980-c9f8-11eb-8211-950dd74f6d55.png)

**Comment by Tejasvigupta** [comment]
Is this still open?

**Comment by maebeam** [comment]
Yes, it would be great to improve the text color and background color here.
